src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -44,6 +44,8 @@ def get_unit_prediction_intervals(self, reporting_units, nonreporting_units, alp
             alpha=alpha,
             beta=self.beta,
         )
+        self.gaussian_bounds_unit = gaussian_model
+        self.conformalization_data_unit = prediction_intervals.conformalization
```

Comment:
these should be defined as `None` in the constructor

Reply:
done (I think as you intended?)